network/servicenetwork/suite_test.go

```diff
@@ -448,11 +442,11 @@ func (s *testSuite) preInitNode(node *networkNode) {
 
 	realKeeper, err := nodenetwork.NewNodeNetwork(cfg.Host, certManager.GetCertificate())
 	s.Require().NoError(err)
-	terminationHandler := &terminationHandler{NodeID: node.id}
+	terminationHandler := termination.NewHandler(serviceNetwork)
```

Comment:
use mock termination handler instead of real, or you will get panic in most tests